Mesh_3/include/CGAL/IO/Complex_3_in_triangulation_3_to_facegraph.h

```diff
@@ -0,0 +1,133 @@
+#ifndef CGAL_COMPLEX_3_IN_TRIANGULATION_3_TO_FACEGRAPH_H
```

Comment:
This file must have license and copyright notices. That is the reason of the [Travis job failure][1].

  [1]: https://travis-ci.org/CGAL/cgal/jobs/255958161